server/client.go

```diff
@@ -4222,6 +4222,7 @@ func (c *client) processServiceImport(si *serviceImport, acc *Account, msg []byt
 	c.pa.reply = nrr
 
 	if changed && c.isMqtt() && c.pa.hdr > 0 {
+		// <>/<> handle QoS2
```

Comment:
Need to understand the purpose of this, but may be no change is needed since QOS2 messages themselves (as opposed to their PUBRELs) are queued up jus like before.